packages/skeleton-svelte/src/lib/components/Accordion/Accordion.test.ts

```diff
@@ -0,0 +1,63 @@
+import { describe, expect, it } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+
+import { Accordion } from '$lib/index.js';
+import AccordionTest from './Accordion.test.svelte';
+
+describe('Accordion', () => {
+	const testId = 'accordion';
+
+	it('Renders the component', () => {
+		render(Accordion, {});
+		const component = screen.getByTestId(testId);
+		expect(component).toBeInTheDocument();
+	});
+
+	for (const prop of ['base', 'padding', 'spaceY', 'rounded', 'width', 'classes']) {
+		it(`Correctly applies the \`${prop}\` prop`, () => {
+			const value = 'bg-green-500';
+			render(Accordion, { [prop]: value });
+			const component = screen.getByTestId(testId);
+			expect(component).toHaveClass(value);
+		});
+	}
+});
+
+describe('Accordion.Item', () => {
+	const testId = 'accordion-item';
+
+	it('Renders the item', () => {
+		render(AccordionTest, {});
+		const component = screen.getAllByTestId(testId)[0];
+		expect(component).toBeInTheDocument();
+	});
+
+	it('Renders all snippets', () => {
+		render(AccordionTest, {});
+		const component = screen.getAllByTestId(testId)[0];
+		expect(component).toHaveTextContent('FooLead');
+		expect(component).toHaveTextContent('FooControl');
+		expect(component).toHaveTextContent('FooPanel');
+	});
+
+	for (const prop of ['base', 'spaceY', 'classes']) {
+		it(`Correctly applies the \`${prop}\` prop`, () => {
+			const value = 'bg-green-500';
+			render(AccordionTest, { childProps: { [prop]: value } });
+			const component = screen.getAllByTestId(testId)[0];
```

Comment:
This should be `getByTestId` right?

Reply:
There's multiple instances of the child components in Accordion.test.svelte. This queries them all, but then we can narrow this down by index. This is useful for some components like Segment Controllers and Tabs, where we can select between the first or second child, which might be active/inactive/etc.

`getByTestId` would probably be equivalent for grabbing the first instance on the page. But I like the explicit that we're selecting on of many.

Reply:
I see, that makes sense 👍